packages/perseus/src/util/extract-perseus-data.ts

```diff
@@ -577,6 +577,47 @@ function injectWidgets(
     return context;
 }
 
+/**
+ * Looks through widgets to identify images that do not have alt text.
+ *
+ * We pass in the fuller perseusRenderer object to allow for the possibility
+ * of expanding to include other img data from content & images in the future.
+ *
+ * @param {PerseusRenderer} perseusRenderer
+ * @returns a stringified list of {widgetId, imageUrl} for imgs that don't have alt text
+ */
+function getImagesWithoutAltData(perseusRenderer: PerseusRenderer): string {
+    if (!perseusRenderer.widgets) {
+        return "";
+    }
+
+    const imgsWithoutAltData: {imgUrl: string; widgetId: string}[] = [];
+
+    keys(perseusRenderer.widgets).forEach((widgetID) => {
+        const widget = perseusRenderer.widgets[widgetID];
```

Comment:
Not a blocker, just a thought

```suggestion
    entries(perseusRenderer.widgets).forEach(([widgetID, widget]) => {
```